test/ETO/EquityToken.js

```diff
@@ -96,32 +99,113 @@ contract("EquityToken", ([admin, nominee, company, broker, ...holders]) => {
       expect(balance).to.be.bignumber.eq(initialBalance);
     });
 
-    it("should overflow on deposit");
+    it("should overflow on deposit", async () => {
+      const initialBalance = new web3.BigNumber(2).pow(256).plus(1);
```

Comment:
you will not overflow this way....
 you need to do it in two steps

1. deposit 2**256-1 in first step - that should work
2. deposit 1 in next - that should revert
check final totalSupply and balance to be 2**256-1

Reply:
That was a version that I had first.